### PR TITLE
ENH/DOC: pydata sphinx theme polishing

### PR DESCRIPTION
**Pull request by tupui**
This is a follow up PR for #13724.

It Address the last comments from reviews and is a starting point for further improvement. Please propose any improvements here (or do a PR on my branch).

Things to check/do:

- [x] ~Icons: NumFOCUS, Tidelift, twitter?~ Should be on the organization website, not here.
- [ ] Polish the landing page
- [ ] Polish `Getting started`
- [ ] Google analytics?
- [x] PDF/zip download links needed? -> link to https://docs.scipy.org/doc/
- [ ] General organization
- [x] Deployment of the doc: verify CI, scipyorg, make, etc. -> seems ok, devdoc is live
- [ ] Developer instructions

### CODE REVIEW

File: doc/source/index.rst
Comment (rgommers) [inline doc/source/index.rst:11]: This is going to be outdated again soon. I suggest replacing the direct links with a single link to https://docs.scipy.org/doc/

Comment (tupuinui) [reply doc/source/index.rst:11]: Thanks, I was not sure what to put here. I wanted to use `|version|` but could not make it work.

Comment (tupui) [reply doc/source/index.rst:11]: Thanks, I was not sure what to put here. I wanted to use `|version|` but could not make it work.

**Comment by justbennet** [comment]
If possible, I suggest that the graphical icons be active links to the documentation as well as the 'To the documentation' text in the grey boxes.  I believe most people would expect that behavior.

**Comment by mdhaber** [comment]
There has been some discussion of analytics at scipy/scipy.org#338.

**Comment by tupui** [comment]
> There has been some discussion of analytics at [scipy/scipy.org#338](https://github.com/scipy/scipy.org/issues/338).

Thanks. Looking at it, there is still no decision. I just saw there are using google analytics in Pandas, hence my question.

**Comment by mdhaber** [comment]
Right, it's still a good question! There were some useful suggestions there, though, e.g. to use something other than Google analytics. Great for the discussion to continue; just thought it's best to see what has been said in the past.

**Comment by tupui** [comment]
@justbennet I added the clickable icons. The solution is not that fancy, I guess it's fine.

**Comment by tupui** [comment]
@jorisvandenbossche proposed to use sphinx-panels instead of the raw html. They are going with it in Pandas. Seems like the good choice here. I will update the PR with the suggestions he made.

**Comment by tupui** [comment]
We are again seeing some timeout in the CI. It's getting concerning.

**Comment by larsoner** [comment]
Looks like there is a conflict. Feel free to ping me for a review when this is ready

**Comment by tupui** [comment]
> Looks like there is a conflict. Feel free to ping me for a review when this is ready

Sorry I did not intentionally ping you (CODEOWNER thing got triggered). But I would still be happy to have your review 😃 

Current CI fails due to a scheduled maintenance from bintray: https://status.bintray.com/ (cc @rgommers in case)

**Comment by larsoner** [comment]
> Sorry I did not intentionally ping you (CODEOWNER thing got triggered).

No problem, it was actually clear from the GitHub email that it was due to CODEOWNER and not you -- I just wanted to chime in to say that I'm happy to review when it's time!

**Comment by tupui** [comment]
Apparently it fails in Latex. I still don't know why.

**Comment by ilayn** [comment]
latexmk wants to run pdflatex twice but somehow something kicks off an xetex run with very old packages. Once I tried to fix that TeX run but always got lost or lost interest probably we have to get to the bottom of it as it is a massive part of doc builds (not so useful either imho).

**Comment by tupui** [comment]
> latexmk wants to run pdflatex twice but somehow something kicks off an xetex run with very old packages. Once I tried to fix that TeX run but always got lost or lost interest probably we have to get to the bottom of it as it is a massive part of doc builds (not so useful either imho).

I am not a user of the PDF doc either. Would be interesting to have some statistics to know if this is really used...

As for now, I can fix the issue by using png images instead of svg if there is no obvious way.

**Comment by tupui** [comment]
Just seeing this now. `mstats` and `qmc` are doubled in the API's section's sidebar.

**Comment by tupui** [comment]
The issue is fixed and to try I set `maxdepth=1`  in the sidebar. Let me know what you think. This is making the html files smaller as intended.

**Comment by larsoner** [comment]
CircleCI is not happy:
```
./scipy-ref.tex:155: Unable to load picture or PDF file 'index_getting_started.
```

**Comment by tupui** [comment]
> CircleCI is not happy:
> 
> ```
> ./scipy-ref.tex:155: Unable to load picture or PDF file 'index_getting_started.
> ```

Thanks for reminding me! I will convert the svg to png (I tested this locally and it fixes the issue).

**Comment by tupui** [comment]
@larsoner now green 🥳

File: doc/source/getting_started.rst
Comment (larsoner) [inline doc/source/getting_started.rst:48]: This appears not to have worked:

https://29663-1460385-gh.circle-artifacts.com/0/html-scipyorg/getting_started.html

Presumably because you killed the `custom-button` CSS below. Probably you want to use `.. link-button::` here, too?

**Review by larsoner — COMMENTED** [review]
Otherwise LGTM. Did you want to take care of other items at the top that aren't checked off yet, or should we merge this once the button is fixed?

File: doc/source/getting_started.rst
Comment (tupui) [reply doc/source/getting_started.rst:48]: Indeed, thanks!

**Comment by tupui** [comment]
> Otherwise LGTM. Did you want to take care of other items at the top that aren't checked off yet, or should we merge this once the button is fixed?

Thanks for reviewing @larsoner. As for the other items, I just listed things I could think about, but I am not sure I am in the best position to propose more than what I did (the quick start guide for instance, not sure if this is enough or if we would want something different. Same for the sidebar, is this what we want in the end?), also my design skills are poor to say the least 😅. I hoped to catch a bit more maintainers here to work on this collectively.

File: doc/source/_templates/sidebar-nav-bs.html
Comment (jorisvandenbossche) [inline doc/source/_templates/sidebar-nav-bs.html:6]: You might want a bigger maxdept for the page that are *not* in the reference section. For example, now "User Guide" -> "Statistics" doesn't show its subsections anymore

File: doc/source/_static/scipy.css
Comment (jorisvandenbossche) [inline doc/source/_static/scipy.css:72]: I would suggest adding `--pst-color-active-navigation: 1, 50, 67;`, that's using the same "warm black" (#013243) as the h1 headers are using. That could give a bit more uniformity in colors.

Comment (tupui) [inline doc/source/_static/scipy.css:72]:
```suggestion
  --pst-color-info: 23, 162, 184;
  --pst-color-active-navigation: 1, 50, 67;
```

File: doc/source/_templates/sidebar-nav-bs.html
Comment (tupui) [inline doc/source/_templates/sidebar-nav-bs.html:4]:
```suggestion
    {{ generate_nav_html("sidebar", maxdepth=2, collapse=True, includehidden=True, titles_only=True) }}
```

File: doc/source/_templates/sidebar-nav-bs.html
Comment (tupui) [inline doc/source/_templates/sidebar-nav-bs.html:6]:
```suggestion
    {{ generate_nav_html("sidebar", maxdepth=2, collapse=True, includehidden=True, titles_only=True) }}
```

Comment (tupui) [reply doc/source/_templates/sidebar-nav-bs.html]: I switched to 2. But IMO this would not be a problem as we have the toc on the right side.

File: doc/source/_templates/sidebar-nav-bs.html
Comment (jorisvandenbossche) [inline doc/source/_templates/sidebar-nav-bs.html:4]:
```suggestion
    {{ generate_nav_html("sidebar", maxdepth=1, collapse=True, includehidden=True, titles_only=True) }}
```

My previous comment was mainly about the user guide, I think you *could* keep it here at a maxdepth of 1 in the reference guide, because it's mainly here that you get such huge sidebars because of all individual docstring pages getting included.

**Review by larsoner — APPROVED** [review]
LGTM +1 for merge

@jorisvandenbossche any other thoughts or comments?

If nobody else has comments I'll try to remember to merge on Monday.

**Comment by larsoner** [comment]
Thanks @tupui !

**Pull request merged**
